support/oidc-discovery-provider/handler.go

```diff
@@ -125,3 +116,15 @@ func (h *Handler) serveKeys(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	http.ServeContent(w, r, "keys", modTime, bytes.NewReader(jwksBytes))
 }
+
+func (h *Handler) verifyHost(host string) error {
+	// Obtain the domain name from the host value, which comes from the
+	// request, or is pulled from the X-Forwarded-Host header (via the
+	// ProxyHeaders middleware). The value may be in host or host:port form.
```

Comment:
This header value may be "attacker controlled". Is there concern in allowing someone to choose it? As far as I can tell it should be ok...

Reply:
Another oops! I confused `X-Forwarded-Host` in this comment for `X-Forwarded-For`, which usually has a different security implication. Nevermind!

Reply:
😬 😌